resources/helm/testing/chart-install-values.yaml

```
@@ -1,6 +1,9 @@
 gateway:
   loglevel: DEBUG
   prefix: /services/dask-gateway
+  env:
+  - name: MY_ENV
+    value: My env
```

Add a test case with a `valueFrom` entry as well.

```suggestion
  env:
  - name: MY_ENV
    value: My env
  - name: MY_CPU_REQUESTS
    valueFrom:
      resourceFieldRef:
        containerName: gateway
        resource: requests.cpu
```